### PR TITLE
Shrink Down to one Start Over Method

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
**Description:**
Uses the one start over method in all places it can be used.

**PR Checklist:**
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

**Review by binh-dam-ibigroup — CHANGES_REQUESTED** [review]
Good fix, could you apply to the "Start Over" button under Trip Tools, please?

**Comment by binh-dam-ibigroup** [comment]
> Good fix, could you apply to the "Start Over" button under Trip Tools, please?

@miles-grant-ibigroup As you do that, let's remove the remaining traces of `reactRouterConfig`.

File: lib/components/narrative/trip-tools.js [hunk 1]
```diff
@@ -218,15 +216,11 @@ const TripTools = ({
...
         break
       case 'START_OVER':
-        const startOverUrl = startOver(
-          reactRouterConfig?.basename,
-          location?.search
-        )
         buttonComponents.push(
           <LinkButton
             Icon={Undo}
+            onClick={() => startOverFromInitialUrl()}
```
Comment (binh-dam-ibigroup) [inline lib/components/narrative/trip-tools.js:222]: Can you just pass the function reference?

**Review by binh-dam-ibigroup — APPROVED** [review]
LGTM

File: lib/components/narrative/trip-tools.js (hunk 1 above)
Comment (miles-grant-ibigroup) [reply lib/components/narrative/trip-tools.js:222]: I tried but it didn't like it for some reason. Something with the import?

**Pull request merged**